### PR TITLE
Update readme to address #9

### PR DESCRIPTION
**Pull request by trevershick**
Fix the donate image.
Add mac example and a notice about javaExecutablePath pointing at the executable, not the path - Addresses #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,24 @@ Linter package must be installed in order to use this plugin. If Linter is not i
...
+
+**Mac Example Settings**
+
+*It is important to note that 'javaExecutablePath' MUST point at the Java executable, not just the path it resides within.*
+```
+  "linter-clojure":
+    clojureExecutablePath: "/Users/theusername/.m2/repository/org/clojure/clojure/1.6.0/clojure-1.6.0.jar"
+    javaExecutablePath: "/usr/bin/java"
+```
 ## Donation
```
Comment (steelbrain) [inline README.md]: You can just drop this part.

Comment (trevershick) [inline README.md]: Do you mean drop the entire 'example settings' section?

Comment (steelbrain) [inline README.md]: I mean the `Donation` part.

Comment (trevershick) [inline README.md]: dropped.

**Comment by steelbrain** [comment]
Thanks

**Pull request merged**